lib/mayaUsd/fileio/transformWriter.cpp

```diff
@@ -156,6 +157,11 @@ void UsdMayaTransformWriter::_ComputeXformOps(
                 for (unsigned int i = 0; i < 3; i++) {
```

Comment:
remove the hardcoded values ?

Reply:
I assume you mean the `3` in the for loop? That line isn't from this PR, its the pre-existing transform writer code

Reply:
My bad - must be going cloud blind in my old age...

Reply:
All good :-)